x-pack/plugins/beats_management/public/components/table/controls.tsx

```diff
@@ -43,6 +60,14 @@ export function ControlBar(props: ControlBarProps) {
     />
   ) : (
     <PrimaryOptions
+      onKueryBarSubmit={onKueryBarSubmit}
```

Comment:
It seems like a lot of these props are being referenced and then passed through so they can be used in other components.

In the re-arch I'm working on, I've compressed a lot of the assignment options fields into an optional prop that is defined in the `table` module and passed along. Do you think it would be useful to employ a similar approach here, since it seems like a lot of these props are unused by the intermediary components?

Reply:
@justinkambic Great question. So the props here are borrowed from whats being used in InfraOps. Perhaps a destruct/spread would be better? That way its passed in how the shared HOC passes in, but we only need to pass through a single prop?

Reply:
👍 that would be good. Can be done in a follow-up, I don't know that we have to halt the changes based on that here, given the timeline for what we want to get merged this week